src/test/java/test/configuration/issue1035/TestclassExample.java

```diff
@@ -0,0 +1,30 @@
+package test.configuration.issue1035;
+
+import org.testng.annotations.AfterClass;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test;
+
+public class TestclassExample {
+
+  @BeforeClass
+  public void beforeClass() throws InterruptedException {
+    printer();
+    Thread.sleep(2000);
+  }
+
+  @Test
+  public void test() {
+  }
+
+  @AfterClass
+  public void afterClass() {
+  }
+
+  private void printer() {
```

Comment:
could be static

Reply:
This method needs access to `hashCode()` which is an instance method. So it cannot be static.